csi.proto

```diff
@@ -228,19 +230,25 @@ message VolumeCapability {
 message CapacityRange {
   // Volume must be at least this big. This field is OPTIONAL.
   // A value of 0 is equal to an unspecified field value.
-  uint64 required_bytes = 1;
+  // NOTE: Plugin must ensure in returning valid **int** values,
+  // **negative** values must **not** be returned.
```

Comment:
How about: `The value of this field must not be negative.` for this one. And same below?

Reply:
same question as above..